Translation Script/Skills.csv

```diff
@@ -284,7 +284,7 @@
 "ダブル手コキ","ダブル手コキ"
 "のダブル手コキ！","'s double footjob!"
 "ダブルペニスキス","ダブルペニスキス"
-"は2人でペニスにキスをした！"," has their penis kissed by two people!"
+"は2人でペニスにキスをした！"," give your penis two kisses at once!"
```

Comment:
I don't think this one really matters, since it's not even in this game, but in the previous game, the attack starts with the names of the attackers, not the player, so I framed it in a more transitive tense.